lib/models/entitiesStore.js

```diff
@@ -28,15 +28,89 @@ var async = require('async'),
     entitiesCollectionName = require('../../config').orionDb.collection,
     myutils = require('../myutils'),
     constants = require('../constants'),
-    logger = require('logops');
+    logger = require('logops'),
+    NGSI = require('ngsijs');
 
 function orionServiceDb(service) {
     return appContext.OrionDb(config.orionDb.prefix + '-' + service);
 }
 
-function findSilentEntities(service, subservice, ruleData, func, callback) {
+function findSilentEntitiesByAPI(service, subservice, ruleData, func, callback) {
+    var context = { op: 'checkNoSignal.findSilentEntitiesByAPI', comp: constants.COMPONENT_NAME };
+
+    var options = {
+        service: service,
+        servicepath: subservice
+    };
+    options.headers = {};
+    // if (token !== null) {
+    //     options.headers[constants.AUTH_HEADER] = token;
+    // }
+    // Add correlator
+    var domain = process.domain;
+    if (domain && domain.context) {
+        options.headers[constants.CORRELATOR_HEADER] = domain.context.corr;
+        // Add other headers
+        if (domain.context.srv && options.headers[constants.SERVICE_HEADER] === undefined) {
+            options.headers[constants.SERVICE_HEADER] = domain.context.srv;
+        }
+        if (domain.context.subsrv && options.headers[constants.SUBSERVICE_HEADER] === undefined) {
+            options.headers[constants.SUBSERVICE_HEADER] = domain.context.subsrv;
+        }
+        if (domain.context.from && options.headers[constants.REALIP_HEADER] === undefined) {
+            options.headers[constants.REALIP_HEADER] = domain.context.from;
+        }
+    }
+    var connection = new NGSI.Connection(config.orion.URL, options);
```

Comment:
It's not necessary to catch any error? are they managed?

Reply:
The callback function is in charge of error handling